src/quote/Quote.spec.ts

```diff
@@ -0,0 +1,60 @@
+import QuoteSchema from './QuoteSchema'
+import IQuote, { ICostBreakdown } from '../types/Quote'
+import Identity from '../identity/Identity'
+import Quote from './Quote'
+
+describe('Claim', () => {
+  const identityAlice = Identity.buildFromURI('//Alice')
+  const invalidCost = { gross: 233, tax: 23.3 } as ICostBreakdown
+  const invalidCostQuoteData = {
+    attesterAddress: identityAlice.address,
+    cTypeHash: '0xa3890sd9f08sg8df9s..',
```

Comment:
Suggestion:
Replace with a regular hash, for clarity.

Reply:
Ok, I have updated.

Reply:
Change request:
cTypeHashes formats are different across tests (Message.spec.tx vs Quote.spec.ts); can you unify these?

Reply:
Updated